src/odemis/acq/milling/test/fibsemos_millmng_test.py

```diff
@@ -46,10 +46,10 @@ class TestOpenFIBSEMMillingManager(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         try:
-            if not openfibsem.OPENFIBSEM_INSTALLED:
-                raise ImportError("OpenFIBSEM package is not installed, please install to enabled milling.")
+            if not fibsemos.FIBSEMOS_INSTALLED:
+                raise ImportError("FIBSEM-OS package is not installed, please install to enabled milling.")
         except ImportError as err:
-            raise unittest.SkipTest(f"Skipping the openfibsem tests, correct libraries "
+            raise unittest.SkipTest(f"Skipping the FIBSEM-OS tests, correct libraries "
```

Comment:
On the README (https://github.com/fibsem-os/fibsem-os), it seems the the spelling is **fibsemOS** (a la Apple). Please adjust the user messages.